examples/jsonschema/example.type.ts

```diff
@@ -78,30 +105,3 @@ export const SelfReference = {
         }
     },
 } as const
-
-export interface Defaults {
```

Comment:
did you check this change?

Reply:
it was stable across several runs now, must have been a change in the OpenAPI